.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 node_modules
+updates
```

Comment:
This could be the location of a git repository of update scripts. See the [Fido Updater scripts](https://github.com/fidoserver/Fido-Updater/tree/master/scripts) for examples.